src/Microsoft.DotNet.Interactive/JavaScriptKernel.cs

```diff
@@ -22,5 +25,20 @@ public JavaScriptKernel() : base(DefaultKernelName)
         {
             return FrontendEnvironment.ExecuteClientScript(command.Code, context);
         }
+
+        public void SetKernelClient(KernelClientBase kernelClient)
```

Comment:
Any reason for this to be public? Any reason for it not to be a constructor parameter?

Reply:
right now we have scenarios where (synapse) there can be a jsclient but not able (yet) to forward commands to the client side kernel